Documentation/README_v2.md

```diff
@@ -0,0 +1,154 @@
+# etcd
```

Comment:
so should I get rid of the link at the top of README.md pointing to the old v2 README too?